auth/src/main/java/com/alfresco/auth/DiscoveryService.kt

```diff
@@ -62,6 +79,31 @@ class DiscoveryService(
         }
     }
 
+    suspend fun isOIDCInstalled(appConfigURL: String): Boolean {
+        val uri = PkceAuthService.discoveryUriWithAuth0(appConfigURL).toString()
+        return withContext(Dispatchers.IO) {
+            try {
+                val client = OkHttpClient.Builder()
+                    .connectTimeout(30, TimeUnit.SECONDS)
+                    .build()
+                val request = Request.Builder()
+                    .url(URL(uri))
+                    .get()
+                    .build()
+                val response = client.newCall(request).execute()
+
+                if (response.code != 200) return@withContext false
+
+                val body = response.body?.string() ?: ""
+                val data = AppConfigDetails.jsonDeserialize(body)
+                return@withContext data?.oauth2?.audience?.isNotBlank() == true
```

Comment:
```suggestion
                return @withContext data?.oauth2?.audience?.isNotBlank() == true
```

Reply:
What are the changes you suggested here?